app/views/clients/rollup/_custom_services.html.haml

```diff
@@ -0,0 +1,7 @@
+:ruby
```

Comment:
<img width="1626" alt="Screenshot 2024-04-10 at 11 26 09 AM" src="https://github.com/greenriver/hmis-warehouse/assets/5333982/babc07c4-ba18-4de1-a475-344e9f5fff86">

@eanders I pushed a fix for Employment Education since those labels were wrong. Just wanted to point out that the Service table is so inconsitent with the others. ("Locations" insetad of "Project", different column order, etc). I didn't touch it though since it looked like it was done relatively recently.

I also pushed relabeling some of the CUstom Service table headers for consistency @martha

Reply:
Thanks @gigxz !  I think we should probably align them all, though as long as the headers match the data below, that's ok for now.